### PR TITLE
Its that time again!

### PR DESCRIPTION
**Pull request by Ryguy6001**
I added some more stuff to the pat, and slap commands, some from tophat, also me. btw tophat needs help fixing his F2A for lathbot.

### CODE REVIEW

**Comment by Ryguy6001** [comment]
Don't pull this! I need to add something in the morning for me.

File: LathBotFront/Commands/ReactionCommands.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 using System.IO;
 using System.Linq;
 using System.Net.Http;
+using System.Runtime;
```
Comment (JulianusIV) [inline LathBotFront/Commands/ReactionCommands.cs:10]: why?

File: LathBotFront/Commands/ReactionCommands.cs [hunk 2]
```diff
@@ -95,7 +103,7 @@ public async Task Repeat(CommandContext ctx, [Description("What the bot should r
         [Description("Parth!")]
         public async Task Parth(CommandContext ctx)
         {
-            await ctx.Channel.SendMessageAsync("Boss man! (even tho the whole senate technically are all equal but i am keeping this for the laughs)");
+            await ctx.Channel.SendMessageAsync("Boss man! (even tho the whole senate technically are all equal but i am keeping this for the laughs, also he is the actual owner)");
```
Comment (JulianusIV) [inline LathBotFront/Commands/ReactionCommands.cs:106]: no need to state this really
```suggestion
            await ctx.Channel.SendMessageAsync("Boss man! (even tho the whole senate technically are all equal but i am keeping this for the laughs)");
```

File: LathBotFront/Commands/ReactionCommands.cs [hunk 3]
```diff
@@ -110,6 +118,10 @@ public async Task Pat(CommandContext ctx, [Description("Who to pat")] DiscordMem
                 await ctx.RespondAsync(new DiscordMessageBuilder().WithStickers([ctx.Guild.Stickers[1013888253272801310]]));
             else if (member.Id == 387325006176059394)//julian
                 await ctx.RespondAsync($"Julian!, you just got pat by {ctx.Member.Mention}");
+            else if (member.Id == 312936081819697154)//Ryguy6001
+                await ctx.RespondAsync($"Ryguy doesn't like getting patted.");
+            else if (member.Id == 671485545288826900)//Tophat
+                await ctx.RespondAsync($"Tophat does not enjoy this, please stop.");//Tophat also requested this for himself.
```
Comment (JulianusIV) [inline LathBotFront/Commands/ReactionCommands.cs:124]: too bad, everyone can be pat, please remove
he can yell at people who try individually

**Review by JulianusIV — CHANGES_REQUESTED** [review]
Please update the source branch, not all commits from the target branch are included yet!

**Comment by JulianusIV** [comment]
also in the future, if you want to publish a pr that is still being worked on you can also use drafts: 
- https://github.blog/news-insights/product-news/introducing-draft-pull-requests/
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request

File: LathBotFront/Commands/ReactionCommands.cs [hunk 4]
```diff
@@ -114,6 +118,8 @@ public async Task Pat(CommandContext ctx, [Description("Who to pat")] DiscordMem
                 await ctx.RespondAsync(new DiscordMessageBuilder().WithStickers([ctx.Guild.Stickers[1013888253272801310]]));
             else if (member.Id == 387325006176059394)//julian
                 await ctx.RespondAsync($"Julian!, you just got pat by {ctx.Member.Mention}");
+            else if (member.Id == 312936081819697154)//Ryguy6001
+                await ctx.RespondAsync($"Ryguy doesn't like getting patted.");
```
Comment (JulianusIV) [inline LathBotFront/Commands/ReactionCommands.cs:122]: Overlooked that earlier sorry, but same for you as with tophat, noone gets to not get pat, just go yelling at people that try to do so

**Comment by Ryguy6001** [comment]
> Please update the source branch, not all commits from the target branch are included yet!

Darn it

**Review by JulianusIV — APPROVED** [review]

**Pull request merged**